prompb/README.md

```diff
@@ -4,6 +4,6 @@ re-compile them when building Prometheus.
 If however you have modified the defs and do need to re-compile, run
 `make proto` from the parent dir.
 
-In order for the script to run, you'll need `protoc` (version 3.12.3) in your
-PATH.
+In order for the [script][../scripts/genproto.sh] to run, you'll need `protoc` (version 3.15.8) in
```

Comment:
```suggestion
In order for the [script](../scripts/genproto.sh]) to run, you'll need `protoc` (version 3.15.8) in
```

Sorry!

Reply:
I think this works in Github-flavored markdown as well? But yeah. Sorry for spending so much time in such a silly PR :P